.github/workflows/release.yml

```diff
@@ -13,10 +13,22 @@ jobs:
   release-please:
     runs-on: ubuntu-latest
     steps:
-      - uses: google-github-actions/release-please-action@v3
+      - id: release
+        uses: google-github-actions/release-please-action@v3
         with:
           release-type: node
           package-name: mongodb
-          pull-request-title-pattern: "chore${scope}: release${component} ${version} [skip-ci]"
-    # Publication steps would be dependent on the output of the above step:
-    # steps.release.outputs.release_created
+          pull-request-title-pattern: 'chore${scope}: release${component} ${version} [skip-ci]'
```

Comment:
Updated, and removed component b/c we don't use that. 
Should we keep the `[skip-ci]` in the final format? I put it there b/c I was running the robot a lot and didn't want to trigger EVG, but I guess it would be good to have the CI green on the PR that triggers the release, thoughts?

Reply:
I actually thought that skip-ci was a great idea for this, and I think we should update our docs automation to do the same; we can run the ci when we're actually closer to releasing by manually removing the tag

Reply:
SGTM